cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/includes/bottom.jsp

```diff
@@ -41,7 +41,7 @@
 <div class="row" style="text-align: center;">
     <br>
     <spring:eval var="createAccountUrl" expression="@casProperties.getProperty('osf.createAccount.url')" />
-    <a id="create-account" href="${createAccountUrl}">Create Account</a>
+    <a id="create-account" href="${createAccountUrl}${osfCampaign.getRegisterUrl()}">Create Account</a>
```

Comment:
"Create Account" at the end of the login page now takes campaigns into consideration.